website/docs/docs/deploy/continuous-deployment.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Continuous deployment in dbt Cloud"
+sidebar_label: "Continuous deployment"
+description: "Learn about continuous deployment (CD) workflows "
+---
+
+To help you improve data transformations and to ensure up-to-date data products, you can run [merge jobs](/docs/deploy/merge-jobs) to implement a continuous deployment (CD) workflow in dbt Cloud. Merge jobs can automatically build modified models for you whenever a pull request (PR) merges, making sure the latest changes are in production (such as modifications to code that generates the tables). You don't need to wait for the next scheduled job to run. 
+
+You can also implement continuous integration (CI) in dbt Cloud, which can help further to reduce the time it takes to push changes to production and improve code quality. To learn more, refer to [Continuous integration in dbt Cloud](/docs/deploy/continuous-integration). 
+
+## How merge jobs work
+
+When you set up merge jobs, dbt Cloud listens for notifications from your Git provider indicating that a PR has been merged. When dbt Cloud receives one of these notifications, it enqueues a new run of the merge job.
+
+You can set up merge jobs to perform one of the following when a PR merges:
+
+| <div style={{width:'350px'}}>Command to run</div> | Usage description |
+| -------- | ----------------- | 
+| `dbt build --select state:modified+` | (Default) Build the modified data with every merge. <br /><br />dbt Cloud builds only the changed data models and anything downstream of it. This ensures that the latest code changes are always pushed to production.  |
```

Comment:
Should we add something like "much like with slim CI, dbt uses deferral to only build the changed data models and anything downstream of it."

Reply:
Not sure how much selling you want to do it docs. But you could also maybe add something about reducing compute costs as a result :)